### ISSUE TITLE
SSL Authentication on APRS-IS - Certificate Expired (but it hasn't)

### ISSUE THREAD

**Issue by hobbes1069** [ge0rg/aprsdroid#203]
I used Google Drive to load my p12 file and it was picked up by APRSDroid and imported correctly.
```
Client certificate not accepted: certificate has expired
```
But my certificate does not expire until March 22nd:

![image](https://user-images.githubusercontent.com/1133256/35019256-7ea463fe-faeb-11e7-8aea-57217e6ed37e.png)

**Comment by ge0rg** [comment]
This looks like an error message from the APRS-IS server you were connecting to (there are two of them right now), maybe it has incorrect date / time settings? Or maybe the .p12 file you used was an old copy?

**Comment by ND7W** [comment]
Been using APRSdroid with no issues for almost one year.  I think it's a great APP.  On June 15, 2019 my log reported that my certificate had expired.  Tried using APRS-IS server northwest.aprs2.net and ssl.aprs2.net and both reported expired certificate.  I opened TQSL and examined the certificate and the date was good through 2021.  Used TQSL to renew the certificate.  I loaded the new certificate into APRSdroid and the log continued to report my certificate had expired.  Uninstalled APRSdroid and again loaded new certificate.  Log continued to report certificate expired.  Uninstalled again and used my APRS-IS passcode.  APRSdroid is working now but I'd really like to know why the certificates are reported as expired?
